src/shared/navgen/nav.cpp

```diff
@@ -0,0 +1,1096 @@
+/*
+   ===========================================================================
+   Copyright (C) 2012 Unvanquished Developers
+
+   This file is part of Daemon source code.
+
+   Daemon source code is free software; you can redistribute it
+   and/or modify it under the terms of the GNU General Public License as
+   published by the Free Software Foundation; either version 2 of the License,
+   or (at your option) any later version.
+
+   Daemon source code is distributed in the hope that it will be
+   useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with Daemon source code; if not, write to the Free Software
+   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+   ===========================================================================
+ */
+// nav.cpp -- navigation mesh generator interface
+
+#include "common/Common.h"
+
+#include <iostream>
+#include <vector>
+#include <queue>
+
+// Filesystem APIs are stupidly declared in different headers
+#ifdef BUILD_CGAME
+#include "engine/client/cg_api.h"
+#else
+#include "sgame/sg_trapcalls.h"
+#endif
+
+#include "engine/qcommon/qcommon.h"
+#include "common/cm/cm_patch.h"
+#include "common/cm/cm_polylib.h"
+#include "common/FileSystem.h"
+#include "navgen.h"
+
+static Log::Logger LOG( VM_STRING_PREFIX "navgen", "", Log::Level::NOTICE );
+
+void UnvContext::doLog(const rcLogCategory /*category*/, const char* msg, const int /*len*/)
+{
+	if ( m_logEnabled )
+	{
+		LOG.Notice(msg);
+	}
+}
+
+static NavgenStatus::Code CodeForFailedDtStatus(dtStatus status)
+{
+	if ( dtStatusDetail( status, DT_OUT_OF_MEMORY ) )
+	{
+		return NavgenStatus::TRANSIENT_FAILURE;
+	}
+	return NavgenStatus::PERMANENT_FAILURE;
+}
+
+static int tileSize = 64;
+
+void NavmeshGenerator::WriteFile() {
```

Comment:
Not sure what you mean. Looks pretty normal to me

Reply:
Usually, the code would put curly braces on their own line, and have the `if ( bla )\n{ \n ...` everywhere. I looked at original file, and it was already a mix of styles, but you seem to have rewritten much of it, too, hence my question.
Again, that would not block the merge, though (would be ridiculous to prevent code improvements for such trivial matter imo, and if that was that important, we should just get some CI stuff the brace/tabs-vs-space/etc code-style).